les_14/hw_les_14/task_6_hw_test.py

```diff
@@ -0,0 +1,67 @@
+# На семинаре 13 был создан проект по работе с
+# пользователями (имя, id, уровень).
+# Напишите 3-7 тестов pytest для данного проекта.
+# Используйте фикстуры.
+
+import re
+import pytest
+from os import path
+from Project import Project
+from User import User
+import Excep as Ex
+
+@pytest.fixture
+def project():
+    return Project.load_users('users.json')
+
+@pytest.fixture
+def admin(project):
+    project.enter('Sam', 114)
+    return project.admin
```

Comment:
Скорее это не фикстура, а именно тест, на работу метода `enter`. Если хотите вернуть фикстуру для теста метода `enter` админа, то просто создайте объект по классу User
```suggestion
def admin(project):
    return User('Sam', 114)
```